pantheon_advanced_page_cache.module

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Implements hook_ENTITY_TYPE_update() for file entities.
+ *
+ * Clears edge cache by file url. If this is not done, changing file contents
+ * does not actually change the file that gets served until the cache times out.
+ */
+function pantheon_advanced_page_cache_file_update(EntityInterface $file) {
+  // Don't try this if we're not in a Pantheon environment.
+  if (function_exists('pantheon_clear_edge_paths')) {
```

Comment:
Use the "happy path" and bail early. No need to nest all the functionality inside a conditional.

```php
if (! function_exists('pantheon_clear_edge_paths')) {
    return;
}
```